docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -554,6 +558,10 @@ You can use only one of the following arguments to connect to your Oracle instan
 
 1. Grant permissions for migration. Use the `yb-voyager-pg-grant-migration-permissions.sql` script (in `/opt/yb-voyager/guardrails-scripts/` or, for brew, check in `$(brew --cellar)/yb-voyager@<voyagerversion>/<voyagerversion>`) to grant the required permissions as follows:
 
+  {{< warning >}}
```

Comment:
@ddhodge, can you take a look at the indentation for the warning to appear under the point? I tried fixing it but couldn’t get it right.